### ISSUE TITLE
Will this work on the Unraid v7.xx betas?

### ISSUE THREAD

**Issue by chilman408** [thor2002ro/unraid_kernel#35]
Hello,

I'm not too savvy with these mods but I'm wondering if the custom kernel will work on Unraid V7 beta?

Also, will these custom kernel allow me to run denuvo games on Steam docker containers?

**Comment by thor2002ro** [comment]
No... Don't try it on 7.x I didn't update it yet

**Comment by rgreco87** [comment]
I'm waiting for it, I'm using an ASUS NUC 14 Pro+ with Intel Core Ultra 9 185H  and I need a more recent Kernel.
Thanks

**Comment by rgreco87** [comment]
> No... Don't try it on 7.x I didn't update it yet

I'm waiting for it, I'm using an ASUS NUC 14 Pro+ with Intel Core Ultra 9 185H and I need a more recent Kernel.
Thanks

**Comment by soultaco83** [comment]
Would love to see this for 7.0 as well. I know it is still in beta for now. There is probably alot that comes into building these kernals. Thank you for your continued work and updating this as you get time

**Comment by sternskr** [comment]
> Would love to see this for 7.0 as well. I know it is still in beta for now. There is probably alot that comes into building these kernals. Thank you for your continued work and updating this as you get time

Yeah honestly, I appreciate this repo so much. I need a newer kernel than exists and this has been saving my ass for a while now. Thanks a ton!
